website/content/docs/concepts/connection-workflows/multi-hop.mdx

```diff
@@ -69,6 +69,63 @@ traffic to a target. Ingress worker filters determine which workers you
 connect with to initiate a session, and egress worker filters determine which
 workers are used to access targets.
 
+## Use HCP-managed workers as ingress workers
+
+Many organizations have strict network policies and prohibit all inbound traffic into their network. In these scenarios, HCP-managed workers can be used as the ingress worker. In order to establish a connection into the network, a self-managed worker configured as an egress worker will initiate an outbound connection to the HCP-managed worker, resulting in a persistent connection. As a result, when end users connect to a target, the end user's connection would hop from the Boundary client to the HCP-managed worker (ingress worker) to the self-managed worker (egress worker) to the target (or other intermediary workers if needed). 
+
+### Configuring HCP-managed workers for ingress 
```

Comment:
```suggestion
### Configure HCP-managed workers for ingress 
```

Reply:
Non-native English speakers sometimes have trouble with -ing words since they can be verbs or nouns in English. So I'd suggest we change this to "configure".